### DISCUSSION TITLE
Optional sorting criteria

### DISCUSSION THREAD

**Discussion by rzelnik** [ransome1/sleek#246] [Ideas]
Multiple sorting criteria is a great feature. Currently it works like this: we have four sorting criteria – Due date, Priority, Projects, Contexts. We can order them by priority, but we are forced to use all four of them. I would also like to be able not to use some sorting criterion – or more exactly, to let the tasks sorted by due date and by priority, and then – all tasks without due date or priority – to let ordered by timeline or by date of creation. This can be done in two ways:

1. to add timeline as a fifth sorting criterion, or
2. to allow addition and removal of sorting criterion, so that we can use any number of sorting criteria (none to all).

Which solution is better for this use case?

**Comment by c-o-m-m-a-n-d-e-r-d-a-t-a** [comment]
Also a way to sort arbitrarily, eg a simple alphabetical sort, which you could use by entering 'line numbers' or similar
